test/dynamo/test_autograd_function.py

```diff
@@ -1076,6 +1058,38 @@ def foo(x):
         foo(torch.randn(2, requires_grad=True))
         self.assertEqual(cnts.frame_count, 1)
 
+    def test_mark_non_differentiable_with_tuple(self):
+        from torch.autograd import Function
+
+        class MyFunction(Function):
+            @staticmethod
+            def forward(ctx, x, y):
+                out1 = x.sin()
+                out2 = y * 2
+                ctx.mark_non_differentiable(out2)
```

Comment:
Can you add another test case where we mark_non_differentiable multiple tensors?